LICENSE

```diff
@@ -3,24 +3,27 @@ typescript-eslint
 Originally extracted from:
 
 TypeScript ESLint Parser
-Copyright JS Foundation and other contributors, https://js.foundation
+BSD 2-Clause License
+
+Copyright (c) 2024, Josh Goldberg ✨
```

Comment:
I didn't meant to indicate in https://github.com/typescript-eslint/typescript-eslint/issues/10280#issuecomment-2455579006 that we should switch typescript-eslint's licenses to be under my name! I was just posting that as an reference.

(the idea of an emoji being in the license is fun though)

Reply:
Alright, I added the bsd-2-clause to the license files for [typescript-estree](https://www.npmjs.com/package/@typescript-eslint/typescript-estree) and [typescript-eslint/plugin](https://www.npmjs.com/package/@typescript-eslint/parser).